docs/content/latest/develop/ecosystem-integrations/spring.md

```diff
@@ -0,0 +1,485 @@
+---
+title: Spring Support for YugabyteDB
+linkTitle: Spring Support for YugabyteDB
+description: Spring Support for YugabyteDB
+aliases:
+  - /develop/ecosystem-integrations/spring/
+menu:
+  latest:
+    identifier: spring
+    parent: ecosystem-integrations
+    weight: 571
+isTocNested: true
+showAsideToc: true
+---
+
+This document describes the Spring Framework support for YugabyteDB.
+
+## Spring Data Cassandra and YCQL
+
+YCQL provides Cassandra wire-compatible query language for client applications to query the YugabyteDB database. YCQL is integrated with the Spring Data Cassandra project and supports POJO-based annotations, data templates, repositories, and so on. 
+
+The following is a non-exhaustive list of supported features:
+
+- Yugabyte Java driver for YCQL 3.n and 4.n drivers.
```

Comment:
@nchandrappa Oracle Style Guide mandated the use of .n for versions (I think the rational was that x has inappropriate connotation in the internet world). For example, in https://docs.oracle.com/middleware/maf210/mobile/install/mafig_setup.htm#MAFIG163 , you'd find the following sentence:
_If your development computer is running either Microsoft Windows 8.n or later, or Mac OS X 10.9.n or later, you have to apply a Hotfix provided by Intel before using the emulator with the Intel HAXM._

Since we don't have our own style guide, the doc team is relying on established doc standards used by professional tech writers for now.